src/utils/system.ts

```diff
@@ -357,27 +356,6 @@ export async function extractZipArchive(
   });
 }
 
-/**
- * Downloads source code of the Github repository and puts it in the specified
- * directory
- *
- * @param owner Repository owner
- * @param repo Repository name
- * @param sha Revision SHA identifier
- * @param directory A directory to extract to
- * @returns A promise that resolves when the sources are ready
- */
-export async function downloadAndExtract(
-  owner: string,
-  repo: string,
-  sha: string,
-  directory: string
-): Promise<void> {
-  const stream = await downloadSources(owner, repo, sha);
-  logger.info(`Extracting sources to ${directory}`);
-  return extractSourcesFromTarStream(stream, directory);
-}
-
```

Comment:
Another sidecar removal—never used in the first place. Yes?

Reply:
Possibly, don't recall this one but I think it was completely unused.